README.md

```diff
@@ -16,8 +16,8 @@ The two primary pyuvsim goals are interferometer simulation accuracy at the leve
 4. Design for scalability across many cpus.
 
 # Installation
-* Bleeding edge: `git clone https://github.com/RadioAstronomySoftwareGroup/pyuvsim`
-<!---* pip: `pip install pyuvsim`-->
+* For simple installation, the latest stable version is available via pip (`pip install pyuvsim`)
+* To install the development version: Clone the repository using `git clone https://github.com/RadioAstronomySoftwareGroup/pyuvsim`, navigate into the pyuvsim directory and run `pip install .` or `python setup.py install`
 * PYUVSim is mainly intended to run on clusters running the linux operating system
```

Comment:
If we're switching to `pyuvsim` instead of PYUVsim... here's another.

Reply:
thanks!